<launch>

  <!-- WEBCAM -->
  <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen" >
    <param name="video_device" value="/dev/video0" />
    <param name="image_width" value="640" />
    <param name="image_height" value="480" />
    <param name="pixel_format" value="yuyv" />
    <param name="camera_frame_id" value="usb_cam" />
    <param name="io_method" value="mmap"/>
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="in_image" to="/usb_cam/image_raw"/>
    <param name="autosize" value="true" />
  </node>

  <!-- SDBACKGROUND -->
  <node pkg="sdbackground" type="sdbackground" name="sdb"
        args=""
    <param name="amplification_factor" value="1" />
    <param name="rate" value="40" />
  </node>
  <node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
    <remap from="image2" to="out_image"/>
    <param name="autosize" value="true" />
  </node>

</launch>

